### PR TITLE
Cleaning of the Particule Accelerator code

### PR DESCRIPTION
**Pull request by Menshin**
Cleaned some old code related to the Particule Accelerator (two almost empty files removed).

`/obj/machinery/particule_accelerator` and `/obj/structure/particule_accelerator` classes should probably be fused together eventually.

### CODE REVIEW

**Comment by KazeEspada** [comment]
Did you get rid readme section?

**Comment by Menshin** [comment]
> Did you get rid readme section?

I'm not sure of what you're referecing here, but if you're speaking of the outdated comments at the top of the file, they're gone.

**Comment by KazeEspada** [comment]
They aren't outdated. That picture still shows how to set up the PA.

**Comment by Menshin** [comment]
Almost everything in these comments is either irrevelent (no longer existing procs, never coded features, etc) or pretty obvious from the code (the part about the icons).

I've readded the map and parts of the PA, but it's rather stuff for the wiki...

File: code/modules/power/singularity/particle_accelerator/particle.dm [hunk 1]
```diff
@@ -40,35 +34,27 @@
...
 
 
 /obj/effect/accelerated_particle/Bumped(atom/A)
 	if(ismob(A))
 		Bump(A)
-	return
 
 
 /obj/effect/accelerated_particle/ex_act(severity, target)
 	loc = null
```
Comment (Shadowlight213) [inline code/modules/power/singularity/particle_accelerator/particle.dm]: Shouldn't this qdel?

Comment (Menshin) [inline code/modules/power/singularity/particle_accelerator/particle.dm]: It should.

**Comment by KazeEspada** [comment]
Hey. The turbine has a map of correct placement in the code and somehow mappers still fucked it up.

**Comment by duncathan** [comment]
My favorite part about the turbine is how it's mapped incorrectly on EVERY MAP

**Pull request merged**